snuba/datasets/processors/search_issues_processor.py

```diff
@@ -327,7 +327,7 @@ def process_message(
             processed = self.process_insert_v1(event, metadata)
         except EventTooOld:
             metrics.increment("event_too_old")
-            raise
+            return None
```

Comment:
@untitaker Wdyt? Unfortunately I needed this so the example payload doesn't fail the processor. It's the same as what errors and other datasets do?

Reply:
I would put EventTooOld behavior behind a snuba setting and turn it on for tests

Reply:
That means we would start DLQing EventTooOld messages everywhere though. I'm not sure we actually want that behaviour since an old event is technically not a message that should be reprocessed.

Reply:
I think all the other processors might doing the right thing and search issues should copy those and drop the message

Reply:
I don't know why issues team decided to add this check but I don't think we should silently lose this data in case of a backlog

Reply:
Yeah that's what I mean - search issues is the odd one out. All of them do this - return None if EventTooOld.

Reply:
We should make it consistent... but I'm very hesitant to change them all over to this standard. We will end up with too many invalid messages in the DLQ that can't be replayed.

Reply:
idk if it's good for dlq replayability to drop old messages either. if the dlq gets replayed really late, we will also drop during replay. that's why i thought a setting would be good

Reply:
It's a fair question - but I don't want us to be changing the behavior of all of our datasets right now just to bump this library. Let's align search issues with what we do everywhere else so we can merge this and revisit that later.